docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  parser:
+    build: .
+    image: node:22.15.0
```

Comment:
If you're using the base image you're not going to have the app from the dockerfile. Gotta use the image from that.